RedlinesProject/App.xaml

```diff
@@ -4,4 +4,18 @@
     xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
     xmlns:local="using:RedlinesProject">
 
+    <Application.Resources>
+        <ResourceDictionary>
+            <Thickness x:Key="GridViewItemRevealBorderThemeThickness">0</Thickness>
+            <ResourceDictionary.ThemeDictionaries>
+                <ResourceDictionary x:Key="Light">
+                    <SolidColorBrush x:Key="MainPageBackground" Color="#fff5f5f5"/>
```

Comment:
What are these for?

Reply:
Main background for the app in MainPage.xaml. The "cards" are using application theme brush background.